src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -15,103 +15,55 @@ description: Touchscreens are available on more and more devices,  from phones u
   </iframe>
 </div>
 
-Touchscreens are available on more and more devices, ranging from phones up to desktop screens. When your users choose to interact with your UI, your app should respond to their touch in intuitive and beautiful ways.
+Touchscreens are available on more and more devices, ranging from phones to
+desktop screens. When your users choose to interact with your UI, your app
+should respond to their touch in intuitive and beautiful ways.
```

Comment:
personally, i'd drop "and beautiful" (and in the matching description), but that's just taste i guess